src/package_manager/apt.ts

```diff
@@ -71,8 +78,10 @@ const distributionSpecificAptDependencies = {
  * @param   packages        list of Debian pacakges to be installed
  * @returns Promise<number> exit code
  */
-export async function runAptGetInstall(packages: string[]): Promise<number> {
-	return utils.exec("sudo", aptCommandLine.concat(packages));
+export async function runAptGetInstall(packages: string[], includeConnext = false): Promise<number> {
+	return utils.exec("sudo",
+                          includeConnext ? aptCommandLineNC.concat(aptCommandLine.concat(packages))
```

Comment:
I'd like to go with a single naming scheme, either `NC` or `connext` - not both.

Probably we don't need to know about `includeConnext` in either `runAptGetInstall` or `installAptDependencies` - this would be a better API with a simple `extra_environment` parameter

Reply:
what do you mean `extra_environemnt` parameter? just making it a global variable or something and set it once we get the `install-connext` variable?

Reply:
I meant that instead of 
```
function runAptGetInstall(packages: string[], includeConnext: bool = false)
```

it would be more flexible as

```
function runAptGetInstall(packages: string[], extraEnvironment: string = "") {
  return utils.exec("sudo", extraEnvironment.concat(aptCommandLine).concat(packages));
```

You could even call it "aptCommandPrefix" - or something that's just not connext-specific, the details should be passed in externally so this generic utility doesn't have to know about this special case